package.json

```diff
@@ -48,7 +48,7 @@
     "typecheck": "tsc --noEmit && yarn run packages:typecheck",
     "plugins:build-bundled": "grafana-toolkit plugin:bundle-managed",
     "watch": "yarn start -d watch,start core:start --watchTheme",
-    "ci:test-frontend": "yarn run test:ci && yarn grafana-toolkit node-version-check",
+    "ci:test-frontend": "yarn run test:ci && npx --yes node-version-check --package",
```

Comment:
@dsotirakis I know this command runs inside drone. Can I assume `npx` will work there? I am guessing yes because it comes as part of npm.